Makefile

```diff
@@ -180,6 +176,12 @@ generate: $(GOPATH)/bin/mockgen
 docker-build:
 	$(BUILD_CMD) -t ${IMG} .
 
+# Build the docker image
+.PHONY: docker-dev-push
+docker-dev-push: build
```

Comment:
Are these changes related to skipping the OpenShift-specific assets? Or are these changes for using the local image registry?